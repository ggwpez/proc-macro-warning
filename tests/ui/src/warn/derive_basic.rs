#[derive(derive::Deprecated)]
struct Test {

}

fn main() {
    let _ = Test { };
}
