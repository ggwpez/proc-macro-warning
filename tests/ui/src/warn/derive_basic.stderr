error: use of deprecated constant `test::_w`:
               It is deprecated to foo.
               Please instead bar.
 --> src/warn/derive_basic.rs:2:1
  |
2 | / struct Test {
3 | |
4 | | }
  | |_^
  |
  = note: `-D deprecated` implied by `-D warnings`
